src/props.shadows.js

```diff
@@ -1,35 +1,36 @@
 const Shadows = {
-  '--shadow-color': '220 3% 15%',
-  '--shadow-strength': '1%',
+  "--shadow-color": "220 3% 15%",
+  "--shadow-strength": "1%",
 
-  '--shadow-color-@media:dark': '220 40% 2%',
-  '--shadow-strength-@media:dark': '25%',
-  
-  '--shadow-1': '0 1px 2px -1px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))',
-  '--shadow-2': `
+  "--shadow-color-@media:dark": "220 40% 2%",
+  "--shadow-strength-@media:dark": "25%",
+
+  "--shadow-1":
+    "0 1px 2px -1px hsl(var(--shadow-color) / calc(var(--shadow-strength) + 9%))",
+  "--shadow-2": `
```

Comment:
all of this is unchanged right? it's just the formatter changed the quotes?

Reply:
yes it's the formatter I have just changed the inner shadows 
and that too i performed a typo by putting fff1 instead hsl